srcpkgs/dmd/template

```diff
@@ -38,6 +38,12 @@ post_extract() {
 	mv phobos-${version} phobos
 }
 
+pre_configure() {
+	case "$XBPS_TARGET_MACHINE" in
```

Comment:
```sh
if [ "$XBPS_TARGET_LIBC" = musl ]; then
...
fi
```

Reply:
But, musl has `vfork(2)` from like `0.8.4`??

Reply:
I explained the reason for this in https://github.com/void-linux/void-packages/pull/25878#discussion_r511674166